src/components/color-picker/color-picker.e2e.ts

```diff
@@ -1060,61 +1142,124 @@ describe("calcite-color-picker", () => {
           });
 
           it("restores previous color value when a nudge key is pressed", async () => {
-            const consistentRgbHsvChannelValue = "0";
-            const initialValue = "#".padEnd(7, consistentRgbHsvChannelValue);
+            const page = await newE2EPage();
+            await page.setContent(`<calcite-color-picker allow-empty></calcite-color-picker>`);
+            const element = await page.find("calcite-color-picker");
+            const initialValue = await element.getProperty("value");
+            const [rgbModeButton, hsvModeButton] = await page.findAll(`calcite-color-picker >>> .${CSS.colorMode}`);
+            const [rInput, gInput, bInput, hInput, sInput, vInput] = await page.findAll(
+              `calcite-color-picker >>> calcite-input.${CSS.channel}`
+            );
 
-            const assertChannelValueNudge = async (page: E2EPage, calciteInput: E2EElement): Promise<void> => {
-              await calciteInput.callMethod("setFocus");
-              await clearAndEnterHexOrChannelValue(page, calciteInput, "");
+            await rgbModeButton.click();
 
-              // using page.waitForChanges as keyboard nudges occur in the next frame
+            // R Input
+            await rInput.callMethod("setFocus");
+            await selectText(rInput);
+            await page.keyboard.press("Delete");
+            await page.keyboard.press("Enter");
+            await page.waitForChanges();
 
-              await page.keyboard.press("ArrowUp");
-              await page.waitForChanges();
-              expect(await calciteInput.getProperty("value")).toBe(consistentRgbHsvChannelValue);
+            expect(await rInput.getProperty("value")).toBe("");
```

Comment:
Why did this change from "0" to "", was that a result of changes to the test or the component code?

Reply:
changes to the component code. The test was always resulting in ""